rfc/rfc017-distributed-network-grpc-v2.md

```diff
@@ -57,6 +65,21 @@ As specified by RFC015, the node MUST authenticate the peer's node DID as follow
 1. Resolve peer's node DID to its corresponding DID document.
 2. Assert that one of the `NutsComm` endpoint's host matches (one of) the `dNSName` SANs in the peer's TLS client certificate.
 
+To prevent bootstrapping issues, a node MAY have an authenticated and an anonymous connection (both sides are unauthenticated) with a peer at the same time. 
```

Comment:
"sides" is ambiguous. It describes the perspective from 1 node, so if the remote node sees the local node as authenticated/unauthenticated is irrelevant for this sentence, I think.

Reply:
Nodes do not accept duplicate connections. To have an authenticated connection next to the bootstrap connection, both sides of the bootstrap connection need to be unauthenticated. If half of the connection is already authenticated, the node holding that connection is going to reject the second connection. This prevents a fully authenticated connection from being established -> no private VC exchange with bootstrap nodes.